tests/installation/partitioning_lvm_thin_provisioning.pm

```diff
@@ -0,0 +1,62 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: LVM thin provisioning setup
+# Maintainer: Martin Loviska <mloviska@suse.com>
+
+use strict;
+use warnings;
+use base 'y2logsstep';
+use testapi;
+use partition_setup qw(create_new_partition_table addpart addlv);
+use version_utils 'is_storage_ng';
+
+sub addvg {
+    my (%args) = @_;
+
+    assert_screen 'expert-partitioner';
+    send_key $cmd{system_view};
+    send_key_until_needlematch('volume_management_feature', 'down');
+    send_key $cmd{addpart};
+    wait_still_screen 2;
+    save_screenshot;
+    send_key 'down';
+    send_key 'ret';
+    assert_screen 'partition-add-volume-group';
+    send_key 'alt-v';
+    type_string $args{name};
+    wait_screen_change { send_key 'tab' } for (0 .. 1);
+    wait_screen_change { send_key 'spc' };
+    assert_screen 'partition-select-first-from-top';
```

Comment:
JFYI, as long as we don't run it yet in textmode, we could use assert_and_click, which is a bit more scalable in case of UI changes.